src/game-support/sleeping-dogs-de.md

```diff
@@ -0,0 +1,12 @@
+# Sleeping Dogs: Definitive Edition
+<!-- script:Aliases [] -->
+
+{{#template ../templates/rating.md status=Platinum installs=Yes opens=Yes}}
```

Comment:
Downgrade to `Silver` due to the minor audio defect.

Reply:
Found a fix for the audio. could we settle for Gold?

...pushing changes now